circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/conf/CircusTrainTransformOptionsConfig.java

```diff
@@ -15,26 +15,15 @@
  */
 package com.hotels.bdp.circustrain.core.conf;
 
-import static com.hotels.bdp.circustrain.core.conf.TableParametersConfig.TABLE_REPLICATION_TABLE_PARAMETERS;
-
-import java.util.Map;
+import static com.hotels.bdp.circustrain.api.conf.TransformOptionsConfig.TRANSFORM_OPTIONS_PROPERTY;
 
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.context.annotation.Configuration;
 
-@Configuration
-@ConfigurationProperties(prefix = TABLE_REPLICATION_TABLE_PARAMETERS)
-public class TableParametersConfig {
-
-  public static final String TABLE_REPLICATION_TABLE_PARAMETERS = "table-properties";
-
-  private Map<String, String> parameters;
+import com.hotels.bdp.circustrain.api.conf.TransformOptionsConfig;
 
-  public Map<String, String> getParameters() {
-    return parameters;
-  }
+@Configuration("transformOptionsConfig")
+@ConfigurationProperties(prefix = TRANSFORM_OPTIONS_PROPERTY)
```

Comment:
Inline the value as called out above.

Reply:
not sure I follow here, do you mean `@ConfigurationProperties(prefix = TRANSFORM_OPTIONS_PROPERTY)` -> `@ConfigurationProperties(TRANSFORM_OPTIONS_PROPERTY)`

or are you referring to `@Configuration("transformOptions")` -> `@Configuration(TRANSFORM_OPTIONS)`

thanks

Reply:
ah I think I follow you, moving the properties from api to core so that they can be referenced here?

Reply:
and I think done